pallets/xor-fee/src/lib.rs

```diff
@@ -714,6 +785,22 @@ pub mod pallet {
             Self::deposit_event(Event::WeightToFeeMultiplierUpdated(new_multiplier));
             Ok(().into())
         }
+
+        #[allow(unused_variables)]
+        #[pallet::call_index(1)]
+        #[pallet::weight(<T as Config>::WeightInfo::set_fee_update_period())]
+        pub fn set_fee_update_period(
+            origin: OriginFor<T>,
+            next_block_number: Option<<T as frame_system::Config>::BlockNumber>,
+        ) -> DispatchResultWithPostInfo {
+            ensure_root(origin)?;
```

Comment:
It checks the democracy or sudo, but you need to check the tech committee credentials.